formal-resume/resume_english.tex

```diff
@@ -22,42 +22,42 @@
+  \homepage{https://mouri.moe} \textperiodcentered\
   \github[MouriNaruto]{https://github.com/MouriNaruto}
 }
 
 \section{Introduction}
 \begin{itemize}
-  \item Hi, I'm Kenji Mouri. But my legal name is Qi Lu which is used only when offline. MouriNaruto, Mouri\_Naruto and Mouri are my typical usernames.
-  \item I'm interested in writing the most compact implementations by using the least syntaxes and third-party libraries. Also, I am desired to make more friends which can talk something about technology.
-  \item I have created and maintained several open-source projects mostly written in C/C++ at night since 2014.
-  \item I'm also a proud Microsoft MVP with Developer Technologies and Windows Development award categories.
-  \item I hope my next job can do something about system or bare-metal related software development and maintenance.
+  \item Hi, I am Kenji Mouri. My legal name is Qi Lu which is used only when offline. MouriNaruto, Mouri\_Naruto and Mouri are my typical usernames.
+  \item I am passionate about developing highly efficient software implementations with minimal syntax and reliance on third-party libraries.
+  \item I have created and maintained several open-source projects written predominately in C and C++ since 2014.
+  \item I am also a proud Microsoft MVP in the Developer Technologies and Windows Development award categories.
+  \item My goal for my next role is system-level software development and maintenance.
 \end{itemize}
 
 \section{Professional Experience}
 
 \datedline{\href{https://live.qq.com}{\textbf{Qi'e TV, Tencent}} (C++ Software Development Engineer)}{Dec 2020 -- Present}
 \begin{itemize}
-  \item Lead the development of the QieLive 2.x Series from scratch to release, adapt for Windows 7 Service Pack 1 or later
-  \item Building customized FFmpeg, MSBuild and Qt6 toolchains for QieLive
+  \item Lead the development of the QieLive 2.x Series from conception to release, adapted for Windows 7 Service Pack 1 or later
+  \item Built customized FFmpeg, MSBuild, and Qt6 toolchains for QieLive
 \end{itemize}
 
 \section{Open Source Experience}
 
 \datedline{\href{https://github.com/M2Team}{\textbf{M2-Team}} (Owner)}{Jun 2015 -- Present}
 \begin{itemize}
   \item \href{https://github.com/M2Team/NanaZip}{NanaZip} - File archiver intended for the modern Windows experience
-  \item \href{https://github.com/M2Team/NanaBox}{NanaBox} - The third-party lightweight out-of-box-experience oriented Hyper-V client
+  \item \href{https://github.com/M2Team/NanaBox}{NanaBox} - A third-party lightweight out-of-box-experience oriented Hyper-V client
   \item \href{https://github.com/M2Team/NanaGet}{NanaGet} - Lightweight file transfer utility based on aria2 and XAML Islands
   \item \href{https://github.com/M2Team/NanaRun}{NanaRun} - Application runtime environment customization utility for Windows
 \end{itemize} 
 
 \datedline{\href{https://github.com/ProjectMile}{\textbf{Project Mile}} (Owner)}{Nov 2020 -- Present}
```

Comment:
What are your goals for Project Mile and how far along are you towards those goals?

Reply:
Project Mile is only the name of my infrastructure libraries.
Mouri Internal Library Essentials (a.k.a. MILE or Mile)

Kenji Mouri

Reply:
I have added the full name for that in https://github.com/MouriNaruto/MouriNaruto/commit/7daf27dcb8a9a57f652df28142aebf1725a182fd because Project Mile is just a codename.

Kenji Mouri